### DISCUSSION TITLE
gluetun + qbittorrent Subnet changes so whitelisting IPs doesn't work

### DISCUSSION THREAD

**Discussion by TBlaikie** [qdm12/gluetun#1977] [Q&A]
Every time that I compose down, then compose up my gluetun +qbittorrent the subnet that qbittorrent sees changes. I am trying to set it up so that I have whitelisted my connection so that I don't have to log in every time. I have had this work a few times by inspecting the gluetun container to find the IP address of the Gateway. Usually it is something like 172.20.0.1. So, in qBittorrent I whitelisted the IPs 172.20.0.0/16, and then I can bypass authentication. But then the next time that I compose down and compose up the gateway changes and I have to change the whitelist again. How can I set the subnet/Gateway to a static value so that it doesn't change on reboot?

**Comment by Unmoved38** [comment]
Give it an static ip?
```
    networks:
      <NetworkName>:
        ipv4_address: 10.0.0.10
 ```

**Reply by samerlinger** [reply]
did you ever figure out a solution to this problem? having the same issue
